### PR TITLE
Confidential Ledger Base Client

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: eng/.docsettings.yml [hunk 1]
```diff
@@ -180,6 +180,7 @@ known_content_issues:
   - ['sdk/textanalytics/azure-ai-textanalytics/swagger/README.md', '#3113']
   - ['sdk/translation/azure-ai-documenttranslator/swagger/README.md', '#3113']
   - ['sdk/webpubsub/azure-messaging-webpubsub/swagger/README.md', '#3113']
+  - ['sdk/confidentialledger/azure-data-confidentialledger/swagger/README.md', '#3113']
```
Comment (srnagar) [inline eng/.docsettings.yml:183]: nit:  this should be alphabetically ordered

File: sdk/confidentialledger/azure-data-confidentialledger/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    .replaceAll("\\w+://", "")
+    .replaceAll("\\..*", "");
+DynamicResponse response = identityServiceClient.getLedgerIdentity(ledgerId).send();
+JsonReader jsonReader = Json.createReader(new StringReader(response.getBody().toString()));
+JsonObject result = jsonReader.readObject();
+String tlsCert = result.getString("ledgerTlsCertificate");
+reactor.netty.http.client.HttpClient reactorClient = reactor.netty.http.client.HttpClient.create()
+    .secure(sslContextSpec -> sslContextSpec.sslContext(SslContextBuilder.forClient()
+        .trustManager(new ByteArrayInputStream(tlsCert.getBytes(StandardCharsets.UTF_8)))));
+HttpClient httpClient = new NettyAsyncHttpClientBuilder(reactorClient).wiretap(true).build();
```
Comment (srnagar) [inline sdk/confidentialledger/azure-data-confidentialledger/README.md:79]: This should be supported in `HttpClientOptions` - @alzimmermsft

File: sdk/confidentialledger/azure-data-confidentialledger/src/main/java/com/azure/data/confidentialledger/ConfidentialLedgerClientBuilder.java [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+                        .build();
+        return httpPipeline;
+    }
+
+    /**
+     * Builds an instance of ConfidentialLedgerBaseClient low level client.
+     *
+     * @return an instance of ConfidentialLedgerBaseClient.
+     */
+    public ConfidentialLedgerBaseClient buildConfidentialLedgerBaseClient() {
```
Comment (srnagar) [inline sdk/confidentialledger/azure-data-confidentialledger/src/main/java/com/azure/data/confidentialledger/ConfidentialLedgerClientBuilder.java:259]: Will the sub-builder be introduced for the next preview?

**Review by srnagar — APPROVED** [review]

File: sdk/confidentialledger/azure-data-confidentialledger/README.md (hunk 2 above)
Comment (alzimmermsft) [reply sdk/confidentialledger/azure-data-confidentialledger/README.md:79]: Is this directed at the SSL context or wiretap?

File: eng/.docsettings.yml (hunk 1 above)
Comment (jianghaolu) [reply eng/.docsettings.yml]: fixed

**Pull request merged**